cppcache/integration/test/FunctionExecutionTest.cpp

```diff
@@ -125,3 +128,33 @@ TEST(FunctionExecutionTest, UnknownFunctionAsyncOnRegion) {
                    .execute("I_Don_t_Exist"),
                FunctionExecutionException);
 }
+
+TEST(DISABLED_FunctionExecutionTest,
```

Comment:
Why are we adding a disabled test?

Reply:
This test reproduces the issue, which includes throwing the _wrong_ exception, aka NotConnectedException.  If I enable the test, I have to change the expected exception to NotConnectedException and change the repro instructions for the server bug (GEODE-6699) to include "change the expected exception to <blah> and rebuild NC tests".  Our thinking was to just leave it like this for the Geode team, I take it you think it should be the other way 'round?

Reply:
I would suggest writing a JIRA to address this once GEODE-6699 is complete.

Reply:
I would also leave a not saying why this is disabled. I would also create a version of the test that asserts the current "correct" behavior that can be removed when the server is corrected.

Reply:
Ima just enable the test and assert the "incorrect" exception, and add a note to the server bug that they'll have to change this line of our test code and rebuild it to repro.  Much less fuss that way.